UPDATING.md

```diff
@@ -420,10 +420,6 @@ indefinitely. This is only available on the command line.
 
 After how much time should an updated DAG be picked up from the filesystem.
 
-#### min_file_parsing_loop_time
```

Comment:
do we need to update the UPDATING.MD given we remove this variable from the config file?

Reply:
Yeah now that 1.10 release has gone out, you are right, thanks. I added it back and made a note that it's disabled.